src/calendar/view/CalendarViewModel.ts

```diff
@@ -444,13 +443,15 @@ export class CalendarViewModel implements EventDragHandlerCallbacks {
 					}
 				} else if (isUpdateForTypeRef(GroupInfoTypeRef, update)) {
 					this._calendarInfos.getAsync().then((calendarInfos) => {
-						const calendarInfo = calendarInfos.get(eventOwnerGroupId) // ensure that it is a GroupInfo update for a calendar group.
+						const calendarInfo = calendarInfos.get(eventOwnerGroupId)
```

Comment:
would it make more sense to only reload if `calendarInfo.groupInfo._id` is the same as updated id? Then we don't need to check group types and such